inc/admin.php

```diff
@@ -857,9 +880,15 @@ function action_builder_content() {
 		);
 		$panels_data            = SiteOrigin_Panels_Styles_Admin::single()->sanitize_all( $panels_data );
 
-		define( 'SITEORIGIN_PANELS_DATABASE_RENDER', true );
+		$GLOBALS[ 'SITEORIGIN_PANELS_DATABASE_RENDER' ] = true;
+
+		do_action( 'siteorigin_panels_setup_database_render', $_POST['post_id'] );
+		SiteOrigin_Panels_Post_Content::single();
+		$post_content = SiteOrigin_Panels_Renderer::single()->render( intval( $_POST['post_id'] ), false, $panels_data );
```

Comment:
This is rendering content but never uses `$post_content`. I assume you wanted to check the output while debugging and forgot to remove this?